test/system/notify-liquidated.test.js

```diff
@@ -0,0 +1,165 @@
+
+    const Auction = await ethers.getContractFactory("Auction")
+
+    const masterAuction = await Auction.deploy()
+    await masterAuction.deployed()
+
+    const CoveragePool = await ethers.getContractFactory("CoveragePool")
+    coveragePool = await CoveragePool.deploy(assetPool.address)
+    await coveragePool.deployed()
+    await assetPool.transferOwnership(coveragePool.address)
+
+    const RiskManagerV1 = await ethers.getContractFactory("RiskManagerV1")
+    riskManagerV1 = await RiskManagerV1.deploy(
+      tbtcToken.address,
+      tbtcDepositTokenAddress,
+      coveragePool.address,
+      signerBondsSwapStrategy.address,
+      masterAuction.address,
+      auctionLength,
+      bondAuctionThreshold
+    )
+    await riskManagerV1.deployed()
+
+    await coveragePool
+      .connect(governance)
+      .approveFirstRiskManager(riskManagerV1.address)
+
+    tbtcDeposit = await ethers.getContractAt("IDeposit", depositAddress)
+
+    bidder = await impersonateAccount(bidderAddress)
+    bidder1 = await impersonateAccount(bidderAddress1)
+  })
+
+  describe("when notified on liquidated auction", () => {
+    let auction
+
+    before(async () => {
+      await tbtcDeposit.notifyRedemptionSignatureTimedOut()
+
+      // The deposit's auction must offer at least 75% of bonds to be accepted
+      // by the risk manager. At starting block, the deposit's auction exposes
+      // 66% we need an additional 9% to pass the risk manager threshold. To get
+      // this part, we need 22870 seconds to elapse. This is because the auction
+      // length is 86400 seconds (24h) and there is 34% of bonds remaining.
+      // So, additional 9% will be offered after 9/34 * 86400s.
+      await increaseTime(22870)
+
+      await riskManagerV1.notifyLiquidation(tbtcDeposit.address)
+
+      const auctionAddress = await riskManagerV1.depositToAuction(
+        tbtcDeposit.address
+      )
+      auction = new ethers.Contract(auctionAddress, Auction.abi, bidder)
+      const bidderTake = lotSize.div(5) // 5 / 5 = 1 TBTC
+      await tbtcToken.connect(bidder).approve(auction.address, bidderTake)
+
+      // bidder takes a partial offer on cov pool auction
+      tx = await auction.takeOffer(bidderTake)
+
+      await tbtcToken.connect(bidder1).approve(tbtcDeposit.address, lotSize)
+      // bidder 1 buys entire deposit outside the coverage pool
+      await tbtcDeposit.connect(bidder1).purchaseSignerBondsAtAuction()
+
+      // notifying a risk manager on deposit being purchased outside the cov pool
+      await riskManagerV1.notifyLiquidated(tbtcDeposit.address)
+    })
+
+    it("should mark a deposit as liquidated", async () => {
+      expect(await tbtcDeposit.currentState()).to.equal(11) // LIQUIDATED
+    })
+
+    it("should early close cov pool auction", async () => {
+      expect(await auction.isOpen()).to.be.false
+    })
+
+    it("should collect surplus from the early closed auction", async () => {
```

Comment:
Can we also assert that `RiskManagerV1.tbtcSurplus` field has a correct value?

Reply:
Sure, done